src/services/ssdi.ts

```diff
@@ -1,197 +1,697 @@
-import fs from 'fs';
+import { promises as fsPromises } from 'fs';
 import path from 'path';
 import { QueryTypes } from 'sequelize';
+// import { z } from 'zod'; // TODO: use for validation of JSON header
 import db from '../models';
+import User from '../policies/user';
+import { auditLogger } from '../logger';
 
-interface Flag {
-  type: string;
+// Constants for readability
+const MAX_POSTGRES_RECORD_LIMIT = 2147483647; // Maximum PostgreSQL supports
```

Comment:
I don't think this is correct https://www.postgresql.org/docs/current/limits.html but honestly I don't imagine there's even a remote chance we'll be able to approach this number with remotely acceptable performance, so it seems very academic to me. What's the intent here? I suspect I'm misunderstanding.

Reply:
https://www.postgresql.org/docs/7.1/sql-createsequence.html#:~:text=The%20optional%20clause%20MAXVALUE%20maxvalue,ascending%20and%20descending%20sequences%2C%20respectively.

Reply:
It looks like after 9.6 the value is considered no max value, https://www.postgresql.org/docs/9.6/sql-createsequence.html